prime-router/src/main/resources/db/migration/V26__convert_action_result_to_text.sql

```diff
@@ -0,0 +1,18 @@
+/*
+ * The Flyway tool applies this migration to create the database.
+ *
+ * Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+ * use VARCHAR(63) for names in organization and schema
+ *
+ * Copy a version of this comment into the next migration
+ *
+ */
+
+
+/*
+ * Changing action_result to TEXT so we can store more information
+ * varchar(2048) just isn't enough
+ */
+ALTER TABLE action
+    ALTER COLUMN action_result TYPE TEXT
+;
```

Comment:
Like I said, it's TEXT now. You're welcome.

Reply:
@jimduff-usds unfortunately now that we're dealing with SOAP services we won't get a good JSON response. We could serialize it from XML to JSON if we want, or add an XML column to hold SOAP responses.

Reply:
We *may* run into performance/timing problems running this in staged / prod. may way to pull it out of flyway.  Depends on how many records are currently there and how postgres handles this underlying data type change.